packages/lesswrong/components/hooks/useUpdateGlossaryPinnedState.ts

```diff
@@ -1,33 +1,49 @@
 import { useUpdate } from "@/lib/crud/withUpdate";
 import { useCurrentUser } from "../common/withUser";
-import { useCallback } from "react";
+import { useCallback, useState } from "react";
 import { useTracking } from "@/lib/analyticsEvents";
+import { useCookiesWithConsent } from "./useCookiesWithConsent";
 
 export function useGlossaryPinnedState() {
   const { captureEvent } = useTracking();
   const currentUser = useCurrentUser();
-  const {mutate: updateUser} = useUpdate({
+  const [cookies, setCookie] = useCookiesWithConsent(['pinnedGlossary']);
```

Comment:
probably this should be using the `PINNED_GLOSSARY_COOKIE`?  (Also `PINNED_GLOSSARY_COOKIE` should be registered as a cookie in `cookies.ts`.)